web/src/components/nomination/LocationInput.vue

```diff
@@ -0,0 +1,106 @@
+<template>
+  <v-row dense>
+    <v-col
+      cols="12"
+      md="6"
+      xs="12"
+    >
+      <v-text-field
+        ref="city"
+        label="City"
+        outlined
+        :value="value.city"
+        @input="updateLocation('city', $event)"
+      />
+    </v-col>
+    <v-col
+      cols="12"
+      md="6"
+      xs="12"
+    >
+      <v-autocomplete
+        ref="prefecture"
+        label="Prefecture"
+        :items="prefectures"
+        item-text="en"
+        outlined
+        :value="value.prefecture"
+        @input="updateLocation('prefecture', $event)"
+      />
+    </v-col>
+  </v-row>
+</template>
+
+<script>
+export default {
+  props: {
+    value: {
+      type: Object,
+      required: true,
+    },
+  },
+  data() {
+    return {
+      locationFields: ['city', 'prefecture'],
+      prefectures: [
```

Comment:
this data is on Airtable `Location` table, isn't it better to fetch from there rather than having it hardcoded?